test/e2e/kubelet.go

```diff
@@ -145,6 +142,26 @@ func createNfsServerPod(c clientset.Interface, config VolumeTestConfig) (*v1.Pod
 	return pod, ip
 }
 
+// Restart the passed-in nfs-server by issuing a `/usr/sbin/rpc.nfsd 1` command in the
+// pod's (only) container. This command changes the number of nfs server threads from
+// (presumably) zero back to 1, and therefore allows nfs to open connections again.
+func restartNfsServer(serverPod *v1.Pod) {
+
+	const startcmd = "/usr/sbin/rpc.nfsd 1"
```

Comment:
There is a `SkipUnlessProviderIs(framework.ProvidersWithSSH...)` call in the test, so these providers are included: {"gce", "gke", "aws", "local"}. Does that cover GCI?